### PR TITLE
Add Material Auto Suggest Input examples

### PR DESCRIPTION
**Pull request by nshahan**
Add examples for selecting all text in Material Input boxes.

### CODE REVIEW

File: lib/src/material_input_demo/material_input_demo.html [hunk 1]
```diff
@@ -25,3 +25,15 @@
...
+  <h3>Select all text in input box</h3>
+  <material-input #manualSelectInput label="Enter some text" type="text"></material-input>
+  <material-button raised (trigger)="selectAllManualInput()">Select all text</material-button>
+  <br>
+  <material-input multiline
+                  #autoSelectInput
+                  ngModel="Click to auto select text"
+                  (focus)="selectAllAutoInput()">
+  </material-input>
+</div>
```
Comment (kevmoo) [inline lib/src/material_input_demo/material_input_demo.html]: missing newline – git likes it

File: lib/src/material_auto_suggest_input_demo/material_auto_suggest_input_demo.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        [displayBottomPanel]="false">
+    </material-input>
+    <material-input
+        floatingLabel label="Empty placeholder"
+        [(ngModel)]="emptyPlaceholder"
+        [displayBottomPanel]="false">
+    </material-input>
+    <material-input
+        floatingLabel label="Leading glyph"
+        [(ngModel)]="leadingGlyph"
```
Comment (kevmoo) [inline lib/src/material_auto_suggest_input_demo/material_auto_suggest_input_demo.html]: I wish this was a search box – unclear what options are available

File: lib/src/material_input_demo/material_input_demo.html (hunk 1 above)
Comment (nshahan) [reply lib/src/material_input_demo/material_input_demo.html]: Done.

File: lib/src/material_auto_suggest_input_demo/material_auto_suggest_input_demo.html (hunk 2 above)
Comment (nshahan) [reply lib/src/material_auto_suggest_input_demo/material_auto_suggest_input_demo.html]: I'll be working on updating the examples soon.

**Pull request merged**